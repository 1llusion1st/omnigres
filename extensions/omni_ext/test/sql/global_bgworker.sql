SELECT omni_ext_test.wait_for_table('global_worker_started');
\d global_worker_started;