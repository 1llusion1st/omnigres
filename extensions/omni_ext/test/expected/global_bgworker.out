SELECT omni_ext_test.wait_for_table('global_worker_started');
 wait_for_table 
----------------
 t
(1 row)

\d global_worker_started;
      Table "public.global_worker_started"
 Column | Type | Collation | Nullable | Default 
--------+------+-----------+----------+---------

